code/_core/datum/damagetype/ranged/magic/pillar.dm

```diff
@@ -0,0 +1,26 @@
+/damagetype/ranged/magic/pillar
+	attack_damage_base = list(
+		BLUNT = 100,
+		HEAT = 100,
+		ARCANE = 100,
+        DARK = 100
+	)
```

Comment:
it has been a hot minute since I last looked at balance for burgerstation-- I feel that 100 in four categories might be a bit? I'll leave someone else to the balance, just wanted to make a comment on it.

Reply:
100 in each is a lot for a base

Reply:
i kinda wanted it to be really strong but i could prolly lower some of them?